src/pretalx/agenda/templates/agenda/talk.html

```diff
@@ -27,6 +27,9 @@ <h3 class="talk-title">
             </a>
         {% endif %}
         <small>
+            <a href="{{ talk.urls.ical }}">
+                <span class="fa fa-calendar"></span>
```

Comment:
The icon just looks like a "here comes the date!"-indicator for the following text and has roughly the same color as the text. Even after skimming this, I needed to go back to the code to find out how to actually get the ical file in the UI. Sadly ical doesn't seem to have any kind of icon, so I don't really know what do to here or where to move the link..

Reply:
Maybe I am just blind and it is fine. Time will show.

Reply:
We could make it a button?

Reply:
I made it a button: 89452e96